pkg/envtest/server.go

```diff
@@ -203,10 +214,13 @@ func (te *Environment) Start() (*rest.Config, error) {
 	}
 
 	log.V(1).Info("installing CRDs")
-	_, err := InstallCRDs(te.Config, CRDInstallOptions{
-		Paths: te.CRDDirectoryPaths,
-		CRDs:  te.CRDs,
-	})
+	if len(te.CRDInstallOptions.CRDs) == 0 {
+		te.CRDInstallOptions.CRDs = te.CRDs
+	}
+	if len(te.CRDInstallOptions.Paths) == 0 {
+		te.CRDInstallOptions.Paths = te.CRDDirectoryPaths
```

Comment:
can't we just append here?

Reply:
Sure, will do. Do we need to filter out "duplicate" paths and CRDs though? In case they same values are passed both directly and via `CRDInstallOptions`?

Reply:
yeah, probably.

Reply:
Just to clarify, "duplicate" CRDs mean they contain the same name right? I don't have to DeepEqual or anything on those CRD objects?

Reply:
identical name is sufficient.  Duplicate name with different contents is an error, so we don't need to try and support that.